janitor/functions/add_columns.py

```diff
@@ -1,12 +1,17 @@
 import pandas_flavor as pf
 
-from janitor.utils import check, deprecated_alias
+from janitor.utils import check, deprecated_alias, refactored_function
 import pandas as pd
 from typing import Union, List, Any, Tuple
 import numpy as np
 
 
+message = "This function will be deprecated in a 1.x release. "
+message += "Kindly use `pd.DataFrame.assign` instead."
+
+
 @pf.register_dataframe_method
+@refactored_function(message=message)
```

Comment:
How does this look?
Remove `message` variable.
```python
@pf.register_dataframe_method
@refactored_function(
    message = (
        "This function will be deprecated in a 1.x release. "
        "Kindly use `pd.DataFrame.assign` instead."
    )
)
```

Reply:
Ahhh you are right ... @thatlittleboy pointed this to me a while back! Forgetful me